lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -469,6 +469,41 @@ void mlir::torch::onnx_c::populateDefaultDomainQtoZ(
                       binder.op, resultType, operand);
                   return success();
                 });
+  patterns.onOp(
+    "Scatter", 9,
```

Comment:
https://github.com/nod-ai/SHARK-TestSuite/tree/main/e2eshark/onnx/operators

lgtm. could you add an e2e test here and run it with your torch-mlir to make sure it's numerically correct? once you do that, ping me and we can merge it.

Reply:
Hi Xida, I added an e2e test, and it is producing numerically identical results. I've submitted my PR here: https://github.com/nod-ai/SHARK-TestSuite/pull/246. Would you be able to review and merge?